pype/plugins/nuke/publish/collect_writes.py

```diff
@@ -113,9 +113,16 @@ def process(self, instance):
                 instance.data["representations"].append(representation)
                 self.log.debug("couldn't collect frames: {}".format(label))
 
+        colorspace = node["colorspace"].value()
+
+        # remove default part of the string
+        if "default (" in colorspace:
+            colorspace = re.sub(r"default.\(|\)", "", colorspace)
```

Comment:
With such a specific string to replace you can just `colorspace.replace("default (", "")`. If it doesnt find it, it wont replace it.

Reply:
I guess you are right except there is also the closing bracket to remove but only in case of `default (` in name. But yeah it could be written differently.